src/main/java/org/alliancegenome/curation_api/jobs/executors/GeneExpressionExecutor.java

```diff
@@ -37,23 +45,61 @@ public void execLoad(BulkLoadFileHistory bulkLoadFileHistory) {
 			}
 			bulkLoadFileDAO.merge(bulkLoadFileHistory.getBulkLoadFile());
 
-			bulkLoadFileHistory.setCount(geneExpressionIngestFmsDTO.getData().size());
+			bulkLoadFileHistory.setCount(ANNOTATIONS, geneExpressionIngestFmsDTO.getData().size());
```

Comment:
This appears to be a duplicate of line 57

Reply:
Duplicate Removed